Makefile.in

```diff
@@ -7,6 +7,7 @@ PREFIX ?= /usr
 BINDIR := $(PREFIX)/bin
 LIBDIR := $(PREFIX)/lib
 DATADIR := $(PREFIX)/share
+MANDIR ?= $(PREFIX)/share/man
```

Comment:
This should be tied to the `DATADIR` as `$(DATADIR)/man`

From FHS, both [`/usr/share/man`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html#usrsharemanManualPages) and [`/usr/local/share/man`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html#usrlocalshare1) are valid and they depend on the `DATADIR`